### PR TITLE
Refactored WorkflowHelper portal_run_id generator to use IdHelper

### PR DESCRIPTION
**Pull request by victorskl**
The `portal_run_id` generator has been pushed up (promoted) to higher order construct `IdHelper`

### CODE REVIEW

**Comment by victorskl** [comment]
@alexiswl FYI

File: data_processors/pipeline/domain/tests/test_workflow.py [hunk 1]
```diff
@@ -16,6 +16,16 @@ class WorkflowDomainUnitTests(PipelineUnitTestCase):
...
 
+    def test_portal_run_id(self):
+        """
+        python manage.py test data_processors.pipeline.domain.tests.test_workflow.WorkflowDomainUnitTests.test_portal_run_id
+        """
+        helper = WorkflowHelper(WorkflowType.BCL_CONVERT)
+        logger.info(helper.get_portal_run_id())
+        self.assertIsNotNone(helper.portal_run_id)
+        self.assertEqual(len(helper.portal_run_id), 16)
+        self.assertEqual(helper.get_portal_run_id(), helper.portal_run_id)
```
Comment (reisingerf) [inline data_processors/pipeline/domain/tests/test_workflow.py:27]: Could add a test that two IDs from two helpers are not the same?
I.e. test that the ID is actually created new each time

**Review by reisingerf — APPROVED** [review]

File: data_processors/pipeline/domain/tests/test_workflow.py (hunk 1 above)
Comment (victorskl) [reply data_processors/pipeline/domain/tests/test_workflow.py:28]: Improved with https://github.com/umccr/data-portal-apis/commit/6232b6f259858a85e4a6838d6a7d3eb7189e37e9

**Comment by victorskl** [comment]
Merging...

**Pull request merged**